taskcat/stacker.py

```diff
@@ -1115,9 +761,10 @@ def stackcreate(self, taskcat_cfg, test_list, sprefix):
                 except TaskCatException:
                     raise
                 except Exception as e:
-                    if self.verbose:
-                        print(PrintMsg.ERROR + str(e))
-                    raise TaskCatException("Cannot launch %s" % self.get_template_file())
+                    raise
+#                    if self.verbose:
```

Comment:
we should move away from committing commented out code. we can always pull code back from version history, it's part of the point.